### ISSUE TITLE
tdate not validating as I'd expect

### ISSUE THREAD

**Issue by chalford** [anweiss/cddl#183]
I'm relatively inexperienced with CBOR & CDDL, so I may be interpreting the spec incorrectly, but I'm having a problem with the `tdate` type.

CDDL file:
```
root = tdate
```

CBOR file (as CBOR-Diag)
```
0("2020-02-11T09:00:00Z")
```
CBOR file (as base64)
```
dDIwMjAtMDItMTFUMDk6MDA6MDBa
```

My understanding was that a `tdate` was defined as `#6.0(tstr)`, which is a `tstr` with a tag of 0. When I run validation with the above files, I get the following validation error:

`[ERROR] Validation of "date.cbor" failed: error validating at cbor location : expected type tdate, got Tag(0, Text("2020-02-11T09:00:00Z"))`

**Comment by anweiss** [comment]
Good catch @chalford! Working on a fix

**Comment by anweiss** [comment]
Fixed in #186.

**Comment by chalford** [comment]
Thank you!
